changelogs/fragments/7389-nmcli-issue-with-creating-a-wifi-bridge-slave.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - nmcli - fix connection.slave-type wired to 'bond' and not with parameter 'slave_type' in case of connection type wifi (https://github.com/ansible-collections/community.general/issues/7389)
```

Comment:
```suggestion
  - nmcli - fix ``connection.slave-type`` wired to ``bond`` and not with parameter ``slave_type`` in case of connection type ``wifi`` (https://github.com/ansible-collections/community.general/issues/7389).
```